client/src/services/psi-cohort.js

```diff
@@ -2,13 +2,11 @@ import store from 'store';
 import { API_URL } from '../constants';
 import { formatCohortDate } from '../utils';
 
-const baseURL = `${API_URL}/api/v1`;
-
 const getCohortAvailAbleSize = (cohort) =>
   cohort.cohort_size - (cohort.participantsCohorts?.length || 0);
 
 export const getPsi = async () => {
-  const response = await fetch(`${baseURL}/psi/with-cohorts`, {
+  const response = await fetch(`${API_URL}/api/v1/psi/with-cohorts`, {
```

Comment:
It's still failing for me even after updating the base url for some reason
only I get a different error
<img width="500" alt="Screen Shot 2022-09-09 at 4 51 01 PM" src="https://user-images.githubusercontent.com/64768811/189459808-d58b8aca-bd48-4a24-ba3c-20f038a6345c.png">